server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -1441,4 +1441,39 @@ public void setMailboxesShouldUpdateMailboxNameWhenRenamingAChildMailbox() {
             .body(ARGUMENTS + ".list", hasSize(1))
             .body(ARGUMENTS + ".list[0].name", equalTo("mySecondBox"));
     }
+
```

Comment:
In this case, the exception will be a MailboxExistExeption without distinction between a target mailbox which would be a system mailbox and a target mailbox which would not be  a system mailbox.